.github/ISSUE_TEMPLATE/user_bug_report.yml

```diff
@@ -0,0 +1,89 @@
+name: User bug report
+description: Create a report to help us improve
+title: '[User reported bug]: '
+labels: C-bug, S-needs-triage
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      value: "I expected to see this happen:
+
+
+      Instead, this happened:
+
+    "
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to reproduce
+      description: Copy and paste the exact commands or code here.
+      placeholder: "Behavior or code sample that causes the bug"
+    validations:
+      required: false
+  - type: textarea
+    id: logs
+    attributes:
+      label: Zebra logs
+      description: Copy and paste the last 100 Zebra log lines or upload the full logs to https://gist.github.com/ and add a link to them here.
+      placeholder: "copy and paste the logs here"
+    validations:
+      required: false
+  - type: checkboxes
+    id: crate
+    attributes:
+      label: In what crates the bug is located, if in any?
```

Comment:
I don't think users will be able to accurately identify the crate causing a bug, so let's delete this list?